clients/include/testing_csrgeam.hpp

```diff
@@ -36,11 +36,14 @@
 using namespace hipsparse;
 using namespace hipsparse_test;
 
+//
+//
+//
```

Comment:
whats this?

Reply:
added for re-triggering clang format, I'll remove them. Otherwise, this is a comment section on top of the routine signature.